### PR TITLE
Initial test for metrics/compute

### PR DESCRIPTION
**Pull request by mdittmer**

### CODE REVIEW

File: metrics/compute/compute_test.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+				&testName,
+				[]metrics.SubTest{},
+			},
+		},
+	}
+	gathered := GatherResultsById(results)
+	assert.Equal(t, len(gathered), 1) // Merged to single TestId: {"A test",""}.
+	for testId, runStatusMap := range gathered {
+		assert.Equal(t, testId, metrics.TestId{"A test", ""})
+		for testRun, testStatus := range runStatusMap {
```
Comment (lukebjerring) [inline metrics/compute/compute_test.go]: Wouldn't it be easier to assert.Equal(t, 2, len(runStatusMap)) and then look up (by key) runA and runB?

File: metrics/compute/compute_test.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			&metrics.TestResults{
+				"A test",
+				"ERROR",
+				&testName,
+				[]metrics.SubTest{},
+			},
+		},
+	}
+	gathered := GatherResultsById(results)
+	assert.Equal(t, len(gathered), 1) // Merged to single TestId: {"A test",""}.
```
Comment (lukebjerring) [inline metrics/compute/compute_test.go]: Swap params @ 1 and 2 around - the API is (t, expected, actual)

File: metrics/compute/compute_test.go (hunk 1 above)
Comment (mdittmer) [reply metrics/compute/compute_test.go]: Sure.

**Comment by rwaldron** [comment]
Landed! a907032c8114bf4821df4c7fe60d202bacf071b1

Tested with: `docker exec -it -u $(id -u $USER):$(id -g $USER) wptd-dev-instance make go_test`

**Pull request closed without merging**